### ISSUE TITLE
musikcube 0.90.0 installed from snap (on non-Ubuntu distros) doesn't start

### ISSUE THREAD

**Issue by adem4ik** [clangen/musikcube#331]
**OS:** Linux 5.4.31-1-MANJARO x86_64 19.0.2 Kyria
**musikcube:** 0.90.0

This is my 1st attempt of installing Musikcube from snap.

**Steps:** 
1) install musikcube from snap
2) start **musikcube.app**

**Result:** 
> ~ >>> sudo snap install musikcube                                                                                                                                                                     
> [sudo] password for adem: 
> musikcube 0.90.0 from clangen installed
> ~ >>> musikcube                                                                                                                                                                                       
> zsh: command not found: musikcube
> ~ >>> musikcube.app                                                                                                                                                                              [127]
> terminate called after throwing an instance of 'std::runtime_error'
>   what():  locale::facet::_S_create_c_locale name not valid
> zsh: abort (core dumped)  musikcube.app
> ~ >>>                                                                                                                                                                                            [134]

**Comment by clangen** [comment]
Interesting, it seems to work fine for me on Ubuntu. This must be some sort of environment-specific issue. Will try to figure out what's going on...

**Comment by siavashserver** [comment]
Same error on Ubuntu 20.04 and musikcube 0.92.1 :
```
❯ musikcube.app 
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
[1]    6237 abort (core dumped)  musikcube.app
```

Edit:
`musikcube_0.93.1_ubuntu_focal_amd64.deb` package works fine.

**Comment by chriztheanvill** [comment]
From Fedora, does not work musikcube with snap

**Comment by clangen** [comment]
Looks like the 0.93.1 release fixed the original issue... @chriztheanvill what happens when you try to start the app? Is there any output to the terminal? I tried on my Fedora 33 VM and it seemed to function fine.

**Comment by chriztheanvill** [comment]
After installing with sudo snap install musikcube, if I try to run musikcube, does not run, with whereis, does not find it.
I've tried with snap refresh, there is no update.

Edit: Sorry, I've did not try to run with 'sudo musikcube.app', then I've tried just 'snap install musikcube' (without sudo), now runs fine.

**Comment by clangen** [comment]
@chriztheanvill what about `musikcube.app`? Also, after I install `snapd` on Fedora I had to reboot before the snaps worked properly -- no idea if that's your issue, just an observation.

**Comment by chriztheanvill** [comment]
No. After install snap, I've tried to run snap install musikcube, and I get an error, wait some seconds, then again run snap and works.
I do not now if my problem was running 'sudo snap install musikcube' instead of using without sudo. Because, right now I run 'snap install musikcube', then musikcube.app and works.

Note: I did not reboot my machine since some months.
Edit: jeje, Now I can not add my music, that stores in '/media/hdd/name/dir'.

Edit 2: I've just rebooted and I have this warning: WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement.
Maybe, just for this, I can not see some dirs.
